### PR TITLE
#337 add CI and nightly release for Windows

### LINKED ISSUE #337: Add nightly for Windows

### PR DESCRIPTION
**Pull request by valleefred**
Closes #337 

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (update of algorithms)
- [ ] main documentation was updated (update of input/output file, update of algorithms)
- [x] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -108,3 +111,65 @@ jobs:
...
           CONTAINER_WORKSPACE: ${{ runner.workspace }}
         run: $CONTAINER_WORKSPACE/sonar/bin/sonar-scanner -Dsonar.host.url=https://sonarcloud.io -Dsonar.cfamily.threads=2
+
+  build-windows:
+    runs-on: windows-2019
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Move workspace
```
Comment (gautierbureau) [inline .github/workflows/ci.yml:121]: maybe not needed for this project if path are not big as in the models compilation of Dynawo...

Comment (valleefred) [reply .github/workflows/ci.yml:121]: Yes but I need deterministic path locations for dynawo-algorithms.cmd to setup correctly environment variables.

Comment (gautierbureau) [reply .github/workflows/ci.yml:121]: not sure what you mean

Comment (valleefred) [reply .github/workflows/ci.yml]: It's ok, i've found another way to do it.

Warning for the future: command line length for unit tests is close to maximum due to path and environment variables!

**Pull request merged**